packages/js/plugins/uri-resolvers/http-resolver/schema.interface.graphql

```diff
@@ -0,0 +1,42 @@
+type Response {
```

Comment:
feels weird having to redefine this when we have it defined here https://github.com/polywrap/http/blob/main/interface/src/schema.graphql but can't think in a better solution rn

Reply:
Let me look at this again. I forget why this was necessary. It seems like it shouldn't be needed.

Reply:
Okay, the reason it's needed is that the version of the CLI in the `dependencies` folder is `latest`, which is 0.9.4. The ENS text record resolver was added after that.